src/ng/compile.js

```diff
@@ -2170,6 +2172,20 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
+    function downloadTemplateDependencies(directive) {
```

Comment:
It looks like this will break if you have two directives and each depend on the other.